CMakeLists.txt

```diff
@@ -1,16 +1,13 @@
 cmake_minimum_required(VERSION 3.5.1)
 include(CheckCCompilerFlag)
 
-project(deflate C ASM)
+project(libdeflate
+    LANGUAGES C ASM
+    VERSION 1.9
+)
 
 set(CMAKE_C_STANDARD 99)
 
-#
```

Comment:
Can you explain why you removed this ?

Reply:
Because it's a standard CMake variable, it's stange to repeat it in the project. A normal CMake user knows that it can controls shared/static with BUILD_SHARED_LIBS.

Reply:
ok